docs/sources/send-data/k8s-monitoring-helm/_index.md

```diff
@@ -0,0 +1,318 @@
+{{< /docs/ignore >}}
+
+To deploy Loki run the following command:
+
+<!-- INTERACTIVE ignore START -->
+```bash
+helm install --values loki-values.yml loki grafana/loki -n meta
+```
+<!-- INTERACTIVE ignore END -->
+
+{{< docs/ignore >}}
+
+```bash
+helm install --values killercoda/loki-values.yml loki grafana/loki -n meta
+```
+{{< /docs/ignore >}}
+
+This command will deploy Loki in the `meta` namespace. The command also includes a `values` file that specifies the configuration for Loki. For more details on how to configure the Loki Helm chart refer to the Loki Helm [documentation](https://grafana.com/docs/loki/<LOKI_VERSION>/setup/install/helm).
+
+## Deploy Grafana
+
+Next we will deploy Grafana to the `meta` namespace. You will use Grafana to visualize the logs stored in Loki. To deploy Grafana run the following command:
+
+```bash
+helm install --values grafana-values.yml grafana grafana/grafana --namespace meta
+```
+
+As before the command also includes a `values` file that specifies the configuration for Grafana. There are two important configuration attributes to take note of:
+
+1. `adminUser` & `adminPassword`: These are the credentials you will use to log in to Grafana. The values are `admin` and `adminadminadmin` respectively. The recommended practice is to either use a Kubernetes secret or allow Grafana to generate a password for you. For more details on how to configure the Grafana Helm chart, refer to the Grafana Helm [documentation](https://grafana.com/docs/grafana/latest/installation/helm/).
+
+2. `datasources`: This section of the configuration lets you define the data sources that Grafana should use. In this tutorial, you will define a Loki data source. The data source is defined as follows:
+
+   ```yaml
+    datasources:
+      datasources.yaml:
+            apiVersion: 1
+            datasources:
+            - name: Loki
+              type: loki
+              access: proxy
+              orgId: 1
+              url: http://loki-gateway.meta.svc.cluster.local:80
+              basicAuth: false
+              isDefault: false
+              version: 1
+              editable: false
+   ```
+  This configuration defines a data source named `Loki` that Grafana will use to query logs stored in Loki. The `url` attribute specifies the URL of the Loki gateway. The Loki gateway is a service that sits in front of the Loki API and provides a single endpoint for ingesting and querying logs. The URL is in the format `http://loki-gateway.<NAMESPACE>.svc.cluster.local:80`. The `loki-gateway` service is created by the Loki Helm chart and is used to query logs stored in Loki. **If you choose to deploy Loki in a different namespace or with a different name, you will need to update the `url` attribute accordingly.**
+
+<!-- INTERACTIVE page step3.md END -->
+
+<!-- INTERACTIVE page step4.md START -->
+
+## Deploy the Kubernetes Monitoring Helm chart
```

Comment:
In Killercoda, for some reason this page opened at the bottom of the page and I had to scroll up.  Not sure if that's a setting, a bug, or because there is so much content on this page?

Reply:
oh that is weird! Let me take a look at that and I will split the stages further